extensions/mssql/package.json

```diff
@@ -513,6 +513,11 @@
           "group": "0_query@1",
           "isDefault": true
         },
+        {
+          "command": "mssql.objectProperties",
```

Comment:
1. Properties should be at the bottom @alanrenmsft Is this something you have on your task list to fix for other types? I see this in the same location for others such as Logins as well

2. Let's remove this for now, you can just keep it locally. But we don't want to enable it until it's complete enough for people to actually start to use (the last thing we want is to have this accidently ship in an unfinished state)

Reply:
1. I'll move it to the bottom of the context menu.
2. @barbaravaldez if the feature is not ready, you can have a feature branch so that you guys can work on it simultaneously.